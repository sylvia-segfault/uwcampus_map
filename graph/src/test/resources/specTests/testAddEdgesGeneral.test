# Create an empty graph
CreateGraph graph4

# Add 4 nodes to graph4
AddNode graph4 node1
AddNode graph4 node2
AddNode graph4 node3
AddNode graph4 node4

# Add 3 edges to graph4
AddEdge graph4 node1 node2 edge1
AddEdge graph4 node1 node3 edge2
AddEdge graph4 node1 node4 edge3