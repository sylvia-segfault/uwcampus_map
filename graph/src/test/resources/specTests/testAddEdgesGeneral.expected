# Create an empty graph
created graph graph4

# Add 4 nodes to graph4
added node node1 to graph4
added node node2 to graph4
added node node3 to graph4
added node node4 to graph4

# Add 3 edges to graph4
added edge edge1 from node1 to node2 in graph4
added edge edge2 from node1 to node3 in graph4
added edge edge3 from node1 to node4 in graph4